sdks/typescript/src/apache_beam/transforms/localParDo.ts

```diff
@@ -0,0 +1,285 @@
+/*
```

Comment:
It looks like this is a copy of parDo, but it's hard to see the diff.

Reply:
im not sure its a good idea to copy the file, instead i think we should extend Pardo.ts to support both.

the question is if we even want both.... this entire PR is more of a proposal, i wanted to get more input on this idea before really implementing it cleanly

Reply:
+1. I was thinking it's just hard to see what's new with the file being copied rather than updated.